### DISCUSSION TITLE
WAZUH VM stuck at booting the kernal in Virtual Box v7.0.6

### DISCUSSION THREAD

**Discussion by espyerx-x** [wazuh/wazuh#21061] [Q&A]
I have imported WAZUH OVA v 4.7.1 in Virtual box. I configured agents for Ubuntu Live server and Windows 10. Agents were working properly. i just restarted my system after installing Openssh server and what i am facing is that VM is stuck at booting the kernal. Now my mouse stucks if i go to the VM  screen and additionally my Capslock remains blinking till i donot shift the TAB using window+TAB. 
Any Help will be appreciated. Thanks in advance.

**Comment by mdiego92** [comment]
Hello @m-raheem,

Are you still having this issue with your VMs? If you check the VM properties, is there any invalid setting detected?
Could you please share with me if there is any related log in the VBox.log file or any message while starting the VM?

Looking forward to your response.
Best regards.
